ngrinder-core/src/test/java/org/ngrinder/AbstractMultiGrinderTestBase.java

```diff
@@ -43,13 +43,13 @@ abstract public class AbstractMultiGrinderTestBase {
 
 	@Before
 	public void agentInit() {
-		agentConfig1 = new AgentConfig.NullAgentConfig(1);
+		agentConfig1 = new AgentConfig.NullAgentConfig(0);
```

Comment:
I think that setting `ngrinder.agent.home` property as `./tmp/agent-home` could derive resource contention. I wanted to fix it but no time. I'll change it if there is a next time.